monkey/monkey_island/cc/ui/src/components/ui-components/InfoPane.js

```diff
@@ -42,11 +60,37 @@ function getSubtitle(props) {
 }
 
 function getBody(props) {
+  let body = [<span key={'body'}>{props.body}</span>];
```

Comment:
Can't you do `(` instead of `[`?

Reply:
Can you declare an array literal with `()`?

Reply:
Sorry, missed that you are actually pushing to array later. Instead of building a variable you should put this directly in the body. And instead of pushing to array, you should conditionally render the warning IMO.

Reply:
Yeah, I played with that, but I couldn't get a syntax that worked well. You can't embed an `if` in the JSX, so I was left with a ternary that looked like `condition ? extra_components : null`. Any suggestions?

Reply:
Maybe something like that?
```suggestion
<Card.Body className={'pane-body'}>
      <span key={'body'}>{props.body}</span>
      {props.warningType !== WarningType.NONE && getWarning(props.warningType)}
</Card.Body>
```